object/organization.go

```diff
@@ -79,9 +79,9 @@ type Organization struct {
 	AccountItems []*AccountItem `xorm:"varchar(5000)" json:"accountItems"`
 }
 
-func GetOrganizationCount(owner, field, value string) (int64, error) {
+func GetOrganizationCount(owner, organization, field, value string) (int64, error) {
 	session := GetSession(owner, -1, -1, field, value, "", "")
-	return session.Count(&Organization{})
+	return session.Where("organization = ? or is_shared = ? ", organization, true).Count(&Organization{})
```

Comment:
why add is_shared in where clause, and i get error message when i use these code in my project. have you test?

![image](https://github.com/user-attachments/assets/8a2c0bd6-b1e9-41d0-9a0f-5d340b7dd98d)

Reply:
fixed this bug.